modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,27 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+In a ruleset's *Governed APIs* page, you can view the list of centrally-governed APIs to which the ruleset applies. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
```

Comment:
```suggestion
* When their APIs fail a ruleset, they can see which other APIs successfully conform to the ruleset so that they can fix their APIs.
```

Reply:
Maybe "...fix their APIs accordingly" to tie this back to seeing the other APIs.